### DISCUSSION TITLE
Docker image cypress/included lastest version is 9.5.1 despite latest Cypress version is 9.5.2

### DISCUSSION THREAD

**Discussion by tit** [cypress-io/cypress#20632] [Docker] [answered]
Hello, I found out that cypress docker image is not publish in docker repo (https://hub.docker.com/r/cypress/included/tags) in lastest version 9.5.2? Is that intentional?
(only 9.5.1 exists)

**Comment by emilyrohrbough** [comment, marked as answer]
Looks like 9.5.2 failed to build & push. https://app.circleci.com/pipelines/github/cypress-io/cypress-docker-images/803/workflows/d38a89e7-4d42-476d-bbba-60428fc7f60f/jobs/37072

**Reply by emilyrohrbough** [reply]
This has been published: https://hub.docker.com/layers/cypress/included/9.5.2/images/sha256-a8f0770a0be0d6486a75b4f8ba151e412adb7f443e192171cc25c36a6ef49afd?context=explore
